style/testing/avoid_let_spec.rb

```diff
@@ -1,23 +1,27 @@
 # BAD
 describe ReportPolicy do
-  let(:user) { User.new(report_ids: [1, 2]) }
-  let(:report) { Report.new(id: 2) }
+  let(:report_id) { 2 }
+  let(:report_policy) do
+    ReportPolicy.new(
+      User.new(report_ids: [1,2]),
+      Report.new(id: report_id)
+    )
+  end
 
   describe "#allowed?" do
+    subject { report_policy.allowed? }
+    
     context "when user has access to report" do
       it "returns true" do
-        policy = ReportPolicy.new(user, report)
-
-        expect(policy).to be_allowed
+        expect(subject).to eql(true)
```

Comment:
sure, and i noticed you guys had the implicit subject recommended elsewhere in the guide just now so i'll change it to that too, figured you guys might not have liked it since it requires the `its` gem but it certainly is nice in these situations.